collector/mongodb_collector.go

```diff
@@ -53,16 +48,11 @@ type MongodbCollectorOpts struct {
 
 func (in *MongodbCollectorOpts) toSessionOps() *shared.MongoSessionOpts {
 	return &shared.MongoSessionOpts{
-		URI:                   in.URI,
-		TLSConnection:         in.TLSConnection,
-		TLSCertificateFile:    in.TLSCertificateFile,
-		TLSPrivateKeyFile:     in.TLSPrivateKeyFile,
-		TLSCaFile:             in.TLSCaFile,
-		TLSHostnameValidation: in.TLSHostnameValidation,
-		PoolLimit:             in.DBPoolLimit,
-		SocketTimeout:         in.SocketTimeout,
-		SyncTimeout:           in.SyncTimeout,
-		AuthentificationDB:    in.AuthentificationDB,
+		URI:                in.URI,
+		PoolLimit:          in.DBPoolLimit,
+		SocketTimeout:      in.SocketTimeout,
```

Comment:
Those things are parts of URI too. If drivers supports then, we should remove them in 0.11 (later).

Reply:
Ok. I've made a ticket.